app/assets/javascripts/discourse/app/static/wizard/components/fields/radio.gjs

```diff
@@ -0,0 +1,72 @@
+import Component from "@glimmer/component";
+import { hash } from "@ember/helper";
+import { on } from "@ember/modifier";
+import { action, set } from "@ember/object";
+import PluginOutlet from "discourse/components/plugin-outlet";
+import concatClass from "discourse/helpers/concat-class";
+import withEventValue from "discourse/helpers/with-event-value";
+import icon from "discourse-common/helpers/d-icon";
+
+export default class Radio extends Component {
+  constructor() {
+    super(...arguments);
+
+    this._setSelected();
+  }
+
+  get field() {
```

Comment:
Im not sure this is useful to do this, in my mind it's often more confusing because you lose track it's an external data and people might then do:

this.field = "xxx"